scripts/simplify_network.py

```diff
@@ -620,6 +766,20 @@ def consense(x):
 
     update_p_nom_max(n)
 
+    n = remove_empty_isolated_nodes(n)
+
+    e_threshold_drop_isolated = cluster_config.get("e_threshold_drop_isolated", False)
+    e_threshold_merge_isolated = cluster_config.get("e_threshold_merge_isolated", False)
+    if e_threshold_drop_isolated:
+        n = drop_isolated_nodes(n, drop_e_threshold=e_threshold_drop_isolated)
```

Comment:
the 1e3 may be placed here and in the call below

Reply:
Revised and removed